### PR TITLE
Fixed headers hierarchy

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR aims at fixing the headers hierarchy. @Fenntasy Pinging you to show how it was easy thanks to the global style, and what we previously did to remove all the headers heritage.

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [x] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/design-system/globals/global-style.tsx [hunk 1]
```diff
@@ -45,20 +45,20 @@ const GlobalStyle = createGlobalStyle<{ theme: DefaultTheme }>`
 	}
 
 	h2 {
+		font-size: ${({ theme }) => theme.fontSizes.h3};
+		font-weight: ${({ theme }) => theme.fontWeights.normal};
+		color: ${({ theme }) => theme.colors.breadcrumbs};
+    margin: 0 0 ${({ theme }) => theme.spaces.s} 0;
+	}
```
Comment (Fuzznimp) [inline src/design-system/globals/global-style.tsx:52]: GitHub issue on lint.

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Review by Oluane — APPROVED** [review]
Nice 👍

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
Task linked: [CU-au05ag [improve] - Heading element not in a sequentially descending order](https://app.clickup.com/t/au05ag)

**Pull request merged**